### PR TITLE
Exercise 9 Workshop MySQL 1

### PR DESCRIPTION
**Pull request by noratop**

### CODE REVIEW

File: exercise-9.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Query OK, 0 rows affected (0.02 sec)
+Records: 0  Duplicates: 0  Warnings: 0
+
+mysql> DESCRIBE Chicken;
++-------------+--------------------------------+------+-----+---------+----------------+
+| Field       | Type                           | Null | Key | Default | Extra          |
++-------------+--------------------------------+------+-----+---------+----------------+
+| id          | int(11)                        | NO   | PRI | NULL    | auto_increment |
+| type        | enum('rooster','hen','chicks') | YES  |     | NULL    |                |
+| comeInOn    | datetime                       | YES  |     | NULL    |                |
```
Comment (nyanofthemoon) [inline exercise-9.txt]: I am assuming birth date.

File: exercise-9.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+| goneOutOn   | datetime                       | YES  |     | NULL    |                |
```
Comment (nyanofthemoon) [inline exercise-9.txt]: I am assuming death date.

**Comment by nyanofthemoon** [comment]
:+1:

File: exercise-9.txt (hunk 1 above)
Comment (noratop) [inline exercise-9.txt]: yes, I just did like there could be another way to *lose* chickens like selling them! or our chickens could have some timeoff leaving the barn for a while! :)

File: exercise-9.txt (hunk 2 above)
Comment (noratop) [inline exercise-9.txt]: Same as above

File: exercise-9.txt (hunk 1 above)
Comment (nyanofthemoon) [inline exercise-9.txt]: Would there be a way to simplify this data model?

**Comment by noratop** [comment]
I replaced the birthdate and deathdate by one single isAlive boolean.

**Pull request closed without merging**